src/TendoPay/Gateway.php

```diff
@@ -47,25 +47,38 @@ function __construct() {
 		$this->init_form_fields();
 		$this->init_settings();
 
-		$this->title             = $this->get_option( self::OPTION_METHOD_TITLE );
+		$this->title = $this->get_option( self::OPTION_METHOD_TITLE );
+		//$this->title             = $this->get_option( self::OPTION_METHOD_TITLE ) . $this->get_logo_tendopay();
 		$this->method_title      = $this->get_option( self::OPTION_METHOD_TITLE );
+		$this->icon              = apply_filters( 'woocommerce_gateway_icon', 'https://placekitten.com/64/64' );
 		$this->description       = $this->get_option( self::OPTION_METHOD_DESC );
 		$this->order_button_text = apply_filters( 'tendopay_order_button_text',
 			__( 'Buy now, pay later with TendoPay', 'tendopay' ) );
 
+
 		$this->maybe_add_payment_initiated_notice();
 		add_action( 'before_woocommerce_pay', [ $this, 'maybe_add_payment_failed_notice' ] );
 
 		$this->view_transaction_url = Constants::get_view_uri_pattern();
 
-		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, array(
+		add_action( 'woocommerce_update_options_payment_gateways_' . $this->id, [
 			$this,
 			'process_admin_options'
-		) );
+		] );
 
 		add_action( 'woocommerce_checkout_init', [ $this, 'maybe_add_outstanding_balance_notice' ] );
 	}
 
+	public function get_logo_tendopay() {
```

Comment:
This method is never used (was used but now the code is commented). Please remove it if you don't use it anymore.